server/schedule/filter/filters.go

```diff
@@ -88,7 +98,7 @@ func filterStoresBy(stores []*core.StoreInfo, keepPred func(*core.StoreInfo) boo
 type Filter interface {
 	// Scope is used to indicate where the filter will act on.
 	Scope() string
-	Type() string
+	Type() filterType
```

Comment:
Why not just keep use string?

Reply:
It is used in array to check the index . if keeps the string type, the counter should be defined as counter []map[string]map[uint64]map[uint64] int , it add one hash calculate .